pkg/spdk/server.go

```diff
@@ -1366,11 +1366,12 @@ func (s *Server) ReplicaBackupStatus(ctx context.Context, req *spdkrpc.BackupSta
 	}
 
 	return &spdkrpc.BackupStatusResponse{
-		Progress:     int32(backup.Progress),
-		BackupUrl:    backup.BackupURL,
-		Error:        backup.Error,
-		SnapshotName: backup.SnapshotName,
-		State:        string(backup.State),
+		Progress:       int32(backup.Progress),
+		BackupUrl:      backup.BackupURL,
+		Error:          backup.Error,
+		SnapshotName:   backup.SnapshotName,
+		State:          string(backup.State),
+		ReplicaAddress: req.ReplicaAddress,
```

Comment:
nit: This approach looks a little strange, as the response field is copied directly from the request. Ideally, the replica address should come from the backup info.

Reply:
Yes,it is a bit weird, but v1 uses the same method.
https://github.com/longhorn/longhorn-engine/blob/master/pkg/sync/backup.go#L133

Reply:
Sounds good. Better to have an issue to track but low priority.

Reply:
Updated. Thank you.